lib/dict.c

```diff
@@ -260,9 +240,8 @@ int rc_read_dictionary (rc_handle *rh, char const *filename)
 			if (sscanf (buffer, "%63s%63s", dummystr, namestr) != 2)
```

Comment:
shouldn't this path (the `$INCLUDE`) only be entered when the dictionary was provided via a real filename (i.e., not via memory)? Otherwise, we will need some testing of that path as well.